configstack/running_module.go

```diff
@@ -286,9 +286,11 @@ type MultiError struct {
 func (err MultiError) Error() string {
 	errorStrings := []string{}
 	for _, err := range err.Errors {
-		errorStrings = append(errorStrings, err.Error())
+		if err.Error() == "exit status 1" {
+            errorStrings = append(errorStrings, err.Error())
+		}
```

Comment:
Again, this affects all commands/args, not just `plan-all --detailed-exit-code`, so this change runs the risk of silently swallowing legitimate errors.

Reply:
I was of the view that since [exit status 2](https://www.terraform.io/docs/commands/plan.html#detailed-exitcode) is not necessarily an error, we should only return it. 
Logging it with the message `Encountered the following errors: Hit multiple errors: exit status 2` is misleading.
What do you think @brikis98 ?